pages/about.tsx

```diff
@@ -74,30 +74,21 @@ export default createPage({
             <p>
               The project is developed in partnership between the NuLawLab and
               scholars at{" "}
-              <a href="https://goodgovproject.com">
-                Boston College Law School's Good Governance Project (GGP)
+              <a href="https://www.bc.edu/bc-web/schools/law.html">
+                Boston College Law School
               </a>{" "}
               and{" "}
               <a href="https://cyber.harvard.edu">
                 Harvard University's Berkman Klein Center for Internet & Society
               </a>
               .
             </p>
-            <h3>BC Good Governance Project</h3>
+            <h3>Boston College Law School</h3>
             <p>
-              GGP is a non-partisan, student-driven initiative at Boston College
-              Law School and the{" "}
-              <a href="https://www.bc.edu/bc-web/centers/clough.html">
-                BC Clough Center for Constitutional Democracy
-              </a>
-              . GGP advocates for faith in democracy, and for democratic
-              structures worth of faith; inclusive, equitable and responsive.
-              Using our Democracy Maps, we aggregate voices from across the
-              state to advance discussion on key governance issues. GGP also
-              provides legal support to pro-democracy organizations and
-              legislators, and host events to galvanize and encourage faith in
-              democracy. Project co-founder Matt Victor is a JD student at BC
-              and the founder of GGP.
+              The BS Law School is an inclusive community of scholars that
```

Comment:
I assume this should read "BC Law School"? going to change that, lmk if not

Reply:
Oops, thank you for catching that!  Please do fix.